challenge.md

```diff
@@ -6,39 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git clone 'url'
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git diff
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git checkout x
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git reset --hard commit-name
 
 5. Suppose that we wanted to look at the third-to-last commit on `master`; what command(s) would we write to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git log
 
 6. OK, we've done all the work we want to do locally. Let's update our fork so that we can make a pull request. What command would we use to update our fork?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git add .
```

Comment:
Well, we do need to `add` and `commit`. However, we really shouldn't ever do `git add .`, since it makes it possible for us to accidentally add things that we shouldn't (for instance, secret keys, or log files). 

To take the changes we've made to our local repo and share them with 'origin' (our fork), we need to run the command `git push origin` (plus, optionally, the specific branch that we'd like to push to).